soccer/radio/SimRadio.cpp

```diff
@@ -120,7 +120,8 @@ void SimRadio::receive() {
                                          : kicker_status_ready);
         rx.set_kicker_voltage(200);
 
-        _reversePackets.push_back(rx);
+        std::lock_guard<std::mutex> lock(_reverse_packets_mutex);
```

Comment:
Wrap just those 2 statements in prackets to scope the mutex lockguard.

Reply:
It'll only run on these two statements. Lock guards go into effect at construct time and go out of effect at scope close, so this is already doing that I think.

Reply:
Ah. Forgot c++ doesn't reorder mutex statements. You're right.